tests/ops/integration_tests/saas/test_onesignal_task.py

```diff
@@ -45,7 +46,7 @@ async def test_non_strict_erasure_request(
 
         player_id = onesignal_erasure_data
         (
-            access_results,
+            _,
```

Comment:
It's just used as a "throwaway" or "don't care" variable. We need to unpack the tuple returned from `non_strict_erasure_request` but since we don't use the `access_results` we can just use `_` to be able to unpack the tuple but indicate that we don't need to use that variable.